script/rubocop-autocorrect.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#
+# Fixes safe cops automatically and creates a commit for each.
+#
+
+if git add --dry-run --all | grep --quiet .; then
+  echo "Dirty working tree. Please start on a fresh branch."
+  exit 1
+fi
+
+# Refresh todo file so that following commits include only related changes:
+bundle exec rubocop --regenerate-todo
+git commit --all --message "Regenerate Rubocop's TODO file"
+
+# Iterate over all safe cops:
+grep "This cop supports safe autocorrection" -A 5 .rubocop_todo.yml\
```

Comment:
The five lines after the autocorrection comment (`-A 5`) is what I found was needed for our todo file. If cops add more comment lines, we may need to increase this value. But a bigger value has the risk that we accidentally include the next cop. So let's hope that this continue to work.

Reply:
I think this should be a comment in the code, that might get lost if the script breaks in the future.

Reply:
Okay, added.